src/GregClient/Utility/Utilities.cs

```diff
@@ -9,6 +9,24 @@
 
 namespace Greg.Utility
 {
+    public static class AppSettingMgr
+    {
+        public static KeyValueConfigurationElement GetItem(String key)
+        {
+            try
+            {
+                var dllPath = new Uri(Assembly.GetExecutingAssembly().GetName().CodeBase).LocalPath;
+                var config = ConfigurationManager.OpenExeConfiguration(dllPath);
+                var enableDebugLogsSetting = config.AppSettings.Settings[key];
+                return enableDebugLogsSetting;
+            }
+            catch
+            {
+                return null;
```

Comment:
can we log something if this fails?

Reply:
We're not logging anything anywhere else....not sure if there is anything useful.

Reply:
I guess we can add console logs...